font_LICENSE.txt

```diff
@@ -1,10 +1,13 @@
 Project                 License                   File
```

Comment:
I'm not certain I updated this file correctly

Reply:
It looks fine to me